std/zig/tokenizer.zig

```diff
@@ -664,27 +666,19 @@ pub const Tokenizer = struct {
                     'x' => {
                         state = State.CharLiteralHexEscape;
                         seen_escape_digits = 0;
-                        expected_escape_digits = 2;
                     },
                     'u' => {
-                        state = State.CharLiteralHexEscape;
-                        seen_escape_digits = 0;
-                        expected_escape_digits = 4;
-                    },
-                    'U' => {
-                        state = State.CharLiteralHexEscape;
-                        seen_escape_digits = 0;
-                        expected_escape_digits = 6;
+                        state = State.CharLiteralUnicodeEscapeSawU;
                     },
                     else => {
                         state = State.CharLiteralEnd;
                     },
                 },
 
                 State.CharLiteralHexEscape => switch (c) {
-                    '0'...'9', 'a'...'z', 'A'...'F' => {
```

Comment:
I assume this was a bug (found when new tests were added)

Reply:
yep. thanks!